src/config/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseUrl" : "http://localhost:8000/"
```

Comment:
We can store these values in a separate environment (.env) file.

- _.env_ contents:
`REACT_APP_API_URL=http://localhost:8000/`

After that, we can get these values in centralized service by accessing `process.env.REACT_APP_API_URL`